db.go

```diff
@@ -0,0 +1,133 @@
+package main
+
+import (
+	"fmt"
+	"github.com/jinzhu/gorm"
+	_ "github.com/jinzhu/gorm/dialects/postgres"
+	"log"
+)
+
+type CertDetail struct {
+	gorm.Model
+	Domain string `gorm:"unique"`
+	Value  string
+}
+
+type DomainAltname struct {
+	gorm.Model
+	Domain string `gorm:"unique"`
+	Value  string
+}
+
+type UserInfo struct {
+	gorm.Model
+	Email string `gorm:"unique"`
+	Value string
+}
+
+func db(dbArgs string) (*gorm.DB, error) {
+	db, err := gorm.Open("postgres", dbArgs)
+	if err != nil {
+		fmt.Println(err)
+		panic("failed to connect to certManager database")
+	}
+
+	return db, err
+}
```

Comment:
You'll never return an error from this function, if err isn't nill, you panic.  You should remove the returned error or the panic.

Reply:
Yes, that make sense.